pyproject.toml

```diff
@@ -37,6 +37,7 @@ doc = [
     "scanpydoc[typehints,theme]>=0.9.4",
     "readthedocs-sphinx-ext",
     "sphinx_copybutton",
+    "dask",
```

Comment:
Should `dask[array,distributed]` and `dask-cuda` be added to the `pyproject-toml` in `dask` section?

Reply:
This is just for building the docs. I think cuml installs all the dask dependencies

Reply:
I think it's generally best practice still to install them directly if you use them.

Reply:
I agree with you. I dont want to run into issue with rapids and the conda installation.